classes/Autocomplete.php

```diff
@@ -9,6 +9,15 @@ class Autocomplete
 
     public $results = [];
 
+    private $option = '';
+
+    private $autocomplete;
+
+    public function __construct()
+    {
+        $this->autocomplete = $this;
+    }
```

Comment:
This isn't making sense to me...can you explain why the class is storing an instance of itself?

Reply:
As there are a mix of static and non static functions I created this to be able to call the instance again. It will probably be removed eventually but didn't have the time to change more.